partiql-eval/src/eval/evaluable.rs

```diff
@@ -505,6 +505,51 @@ impl Evaluable for EvalFilter {
     }
 }
 
+/// Represents an evaluation `Having` operator; for an input bag of binding tuples the `Having`
+/// operator filters out the binding tuples that does not meet the condition expressed as `expr`,
+/// e.g. `a = 10` in `HAVING a = 10` expression.
+#[derive(Debug)]
+pub struct EvalHaving {
+    pub expr: Box<dyn EvalExpr>,
+    pub input: Option<Value>,
+}
+
+impl EvalHaving {
+    pub fn new(expr: Box<dyn EvalExpr>) -> Self {
+        EvalHaving { expr, input: None }
+    }
+
+    #[inline]
+    fn eval_having(&self, bindings: &Tuple, ctx: &dyn EvalContext) -> bool {
+        let result = self.expr.evaluate(bindings, ctx);
+        match result.as_ref() {
+            Boolean(bool_val) => *bool_val,
+            // Alike SQL, when the expression of the HAVING clause expression evaluates to
+            // absent value or a value that is not a Boolean, PartiQL eliminates the corresponding
+            // binding. PartiQL Specification August 1, August 1, 2019 Draft, Section 11.1.
```

Comment:
```suggestion
            // binding. PartiQL Specification August 1, 2019 Draft, Section 11.1.
```

Reply:
Ah, nice catch. There was the same typo for `EvalFilter`. Will fix in next commit